content/lessons/genesis/genesis.tsx

```diff
@@ -39,15 +42,19 @@ export default function Genesis1() {
         <Title>Find block 0</Title>
 
         <Text className="text-lg">
-          Let’s find the very first block in the Bitcoin blockchain. Go to a
-          <Tooltip
-            content="We recommend https://blockstream.info/"
-            href="https://blockstream.info/"
-          >
-            block explorer
-          </Tooltip>
-          and search for ’0’. This will take you to the block at height 0. As
-          you can see, this block only contains one transaction.
+          <FormattedMessage
```

Comment:
Now that I'm revisiting this PR I think there is a lot of complexity here just to render out some strings. I touched upon this in another comment but it would be great if we can abstract away the `react-intl` specific stuff somehow so people don't have to get familiar with the library.

If we could get this to more closely resemble what I have in [my solution](https://github.com/saving-satoshi/saving-satoshi/pull/122/files#diff-adb4d3d69d8ccad9528f9d28420f1e3a7bc4558ae1baf51b980924a6e707c19eR42), that'd be great.

Reply:
Might be worth spiking on something like https://www.i18next.com/ instead of adding abstractions to react-intl. IMO the main benefit of using something like react-intl is using all the components they provide for rendering the strings. i18next more resembles what you wrote and I think it also provides easy use of rendering children like the Tooltip components.

Reply:
Or we could just use try out what you made. Might be good enough.

Reply:
But also keep in mind that most of the time the translation rendering code doesn't look like this. This is a more complex example of rendering a nested component that also has a translated prop and children.

Reply:
Another benefit of i18next is you can easily plug into their platform http://locize.com.